src/crimson/os/seastore/omap_manager/btree/omap_btree_node.h

```diff
@@ -116,17 +116,14 @@ struct OMapNode : LogicalChildNode {
 
   virtual ~OMapNode() = default;
 
-  void init_range(std::string begin, std::string end) {
-    if (!this->end.empty()) {
-      // this is a duplicated init.
```

Comment:
It seems omap has duplicated calls to `init_range()` for reasons other than `maybe_init()`.

Reply:
Tests look good now.